sos/report/plugins/__init__.py

```diff
@@ -3407,6 +3411,22 @@ def setup_verify(self):
         if verify_cmd:
             self.add_cmd_output(verify_cmd)
 
+    def container_path_exists(self, path, container):
+        """Check if a path exists inside a container before
+        collecting a dir listing
+
+        :param path:    The canonical path for a specific file/directory
+                        in a container
+        :type path:     ``str``
+
+        :param container: The container where to check for the path
+        :type container: ``str``
+
+        :returns:       True if the path exists in the container, else False
+        :rtype:         ``bool``
+        """
+        return self.exec_cmd(f"test -e {path}", container=container)
```

Comment:
I'm sure there are better ways to do this, but I didn't find any way in our code already, so I had to come up with this 'dirty' trick. 
As always, open to suggestions to do this in a better way :)